packages/renderer/src/lib/preferences/PreferencesConnectionsEmptyRendering.svelte

```diff
@@ -0,0 +1,18 @@
+<script lang="ts">
+import LineWithLinks from './LineWithLinks.svelte';
+
+export let message: string;
+export let hidden: boolean;
```

Comment:
This is dynamic, because if provider sets empty connections message it should be hidden when first connection created. I just mimicked the EmptyScreen component which has the hidden attribute.